specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2024-01-01-preview/mfe.json

```diff
@@ -2228,6 +2228,78 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/registries/{registryName}/datareferences/{name}/versions/{version}": {
+      "post": {
```

Comment:
uri is invalid for a post action

/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/registries/{registryName}/datareferences/{name}/versions/{version}/getBlobReferenceUri?

Reply:
I understand the policy requires us to add an ActionName at the end of the uri.
But by changing that will bring in a break change.
@rkmanda, would it be better if we keep it the same as our previous version 2023-10-01?

Reply:
This seems to be a new API u are introducing, why would changing this become breaking?

Reply:
looks like u changed it

Reply:
Yes, but I want to get your feedback on it.
This is not a new API, we are preparing this new version 2024-01-01-preview now.
I need to bring in the changes in previous version 2023-10-01.
Previous changes PR is here: https://github.com/Azure/azure-rest-api-specs/pull/25619

Reply:
so my ask is, is it okay if we don't make this change to keep it identical to our latest version 2023-10-01?

Reply:
As discussed with Zizhou, we will fix this issue in the future version with more plenty of time. We will bypass this policy this time to avoid the breaking changes.
A work item is created to track the fix: https://dev.azure.com/msdata/Vienna/_workitems/edit/2803196